### PR TITLE
chore(macros/XsltRef) update zh-CN translation

### PR DESCRIPTION
**Pull request by skyclouds2001**
## Summary

update translation for {{XsltRef}}

### Solution

---

### Before

none

### After

none

---

## How did you test this change?

none, as this macros has no usage in zh-CN translations

---

/cc @yin1999 @JasonLamv-t @jasonren0403 

part of the https://github.com/mdn/translated-content/issues/13881

BEGIN_COMMIT_OVERRIDE
chore(macros/XsltRef): update zh-CN translation
END_COMMIT_OVERRIDE

### CODE REVIEW

File: kumascript/macros/XsltRef.ejs [hunk 1]
```diff
@@ -16,6 +16,8 @@
         case "ko":
             html = '<b><a href="/ko/docs/Web/XSLT/Transforming_XML_with_XSLT/The_Netscape_XSLT_XPath_Reference">XSLT/XPath 참고 문서</a></b>: <a href="/ko/docs/Web/XSLT/Element">XSLT 요소</a>, <a href="/ko/docs/Web/XPath/Functions">XPath 함수</a>, <a href="/ko/docs/Web/XPath/Axes">XPath 축</a>';
             break;
+        case "zh-CN":
+            html = '<b><a href="/zh-CN/docs/Web/XSLT/Transforming_XML_with_XSLT/The_Netscape_XSLT_XPath_Reference">XSLT/XPath 参考</a></b>: <a href="/zh-CN/docs/Web/XSLT/Element">XSLT 元素</a>, <a href="/zh-CN/docs/Web/EXSLT">EXSLT 函数</a>, <a href="/zh-CN/docs/Web/XPath/Functions">XPath 函数</a>, <a href="/zh-CN/docs/Web/XPath/Axes">XPath 轴</a>';
```
Comment (yin1999) [inline kumascript/macros/XsltRef.ejs:20]:
```suggestion
            html = '<b><a href="/zh-CN/docs/Web/XSLT/Transforming_XML_with_XSLT/The_Netscape_XSLT_XPath_Reference">XSLT/XPath 参考</a></b>：<a href="/zh-CN/docs/Web/XSLT/Element">XSLT 元素</a>、<a href="/zh-CN/docs/Web/EXSLT">EXSLT 函数</a>、<a href="/zh-CN/docs/Web/XPath/Functions">XPath 函数</a>、<a href="/zh-CN/docs/Web/XPath/Axes">XPath 轴</a>';
```

**Review by yin1999 — COMMENTED** [review]
nit

**Review by yin1999 — APPROVED** [review]
Thank you @skyclouds2001

**Comment by yin1999** [comment]
Screenshot of `/zh-CN/docs/Web/EXSLT/regexp`:

image

**Review by caugner — APPROVED** [review]

**Pull request merged**